lib/Host.js

```diff
@@ -104,7 +104,7 @@ module.exports = function (ts) {
 	Host.prototype.getDefaultLibFileName = function () {
 		var libPath = path.dirname(ts.sys.getExecutingFilePath());
 		var libFile = ts.getDefaultLibFileName({ target: this.languageVersion });
-		return path.join(libPath, libFile);
```

Comment:
![](https://cdn.meme.am/instances/500x/71599399.jpg)
:metal:  Thanks :rose: 

![](https://cdn.meme.am/instances/500x/71599418.jpg)

True story : https://github.com/alm-tools/alm/blob/ca90b703424e18390b0756784f05c2ea4c1a5702/src/server/utils/fsu.ts#L2

![](http://i2.kym-cdn.com/entries/icons/original/000/006/216/7nTnr.png)

/cc @johnnyreilly @blakeembrey @nycdotnet cause you deserve a laugh now and again :heart: